src/Helpers/SAMLHelper.php

```diff
@@ -75,7 +75,8 @@ public function redirect(RequestHandler $requestHandler = null, HTTPRequest $req
         $additionalGetQueryParams = $this->getAdditionalGETQueryParameters();
 
         try {
-            $auth->login(Director::absoluteBaseURL() . 'saml/', $additionalGetQueryParams);
```

Comment:
By the way, this usage of login's first parameter is wrong, it results in a URL like `https://mysite.orgsaml`, i.e. it's missing the slash. The bug didn't manifest as an issue, because RelayState isn't used anyway.